app/build.gradle

```diff
@@ -76,6 +76,9 @@ dependencies {
     compile 'com.squareup.retrofit2:converter-gson:2.1.0'
```

Comment:
If we are using moshi, we can remove gson.

Reply:
@chiragw15 Is it ok if i resolve conflict from github interface or should i do it from git?

Reply:
Anyone is fine.